mkosi/__init__.py

```diff
@@ -8340,7 +8343,8 @@ def run_verb(raw: argparse.Namespace) -> None:
 
         save_manifest(args, manifest)
 
-        print_output_size(args)
+        if not args.skip_final_phase:
+            print_output_size(args)
```

Comment:
We should modify `print_output_size()` to not do anything if `args.output` doesn't exist

Reply:
Just for the directory/subvolume case?

Reply:
All cases